sklearn/feature_selection/_rfe.py

```diff
@@ -155,9 +160,17 @@ def _fit(self, X, y, step_score=None):
         n_features = X.shape[1]
         if self.n_features_to_select is None:
             n_features_to_select = n_features // 2
+        elif type(self.n_features_to_select).__name__=='float':
+            if 0.0 < self.n_features_to_select < 1.0:
+                n_features_to_select = int(n_features *
```

Comment:
Let's use `ceil` from the `math` module here, to ensure that `n_features_to_select` is never 0.